sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -22,12 +30,18 @@ export default {
   setup(props: any) {
     // Hooks
     const initialData = ref({});
+    const confirmAssessment = async () => {
+      await ApplicationService.shared.confirmationOfAssessment(
```

Comment:
Please consider adding some toast notification for success or error. It is probably not part of the ticket but it will help us troubleshooting.